### DISCUSSION TITLE
Configuring RDS Proxy thresholds

### DISCUSSION THREAD

**Discussion by qmitchell-aa** [prisma/prisma#23518] [Q&A] [answered]
### Question

AWS RDS Proxy has connection thresholds, but prisma appears to not support setting them.

My question is, does prisma have defaults somewhere internally that already adhere to these requirements?  
* Connection max life
* Idle connection timeout

If so, it would be helpful to point to the line of code where these defaults exist so that they could be configured by the prisma client. 

https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/rds-proxy-managing.html#rds-proxy-connection-life

> Client connection max life: RDS Proxy enforces a maximum life of client connections of 24 hours. This value is not configurable. Configure your pool with a maximum connection life less than 24 hours to avoid unexpected client connection drops.
> 
> Client connection idle timeout: RDS Proxy enforces a maximum idle time for client connections. Configure your pool with an idle connection timeout of a value lower than your client connection idle timeout setting for RDS Proxy to avoid unexpected connection drops.

**Comment by janpio** [comment, marked as answer]
These are currently not surfaced to be configured.
Please open a feature request for this, AWS RDS Proxy suggesting this is a good enough use case for us to consider adding this.

**Reply by qmitchell-aa** [reply]
Thanks. Done, https://github.com/prisma/prisma/issues/23529
